packages/navigator/src/browser/navigator-model.ts

```diff
@@ -186,4 +186,12 @@ export class FileNavigatorModel extends FileTreeModel {
                 node1.id.length >= node2.id.length ? node1 : node2
             ) : undefined;
     }
+
+    async collapseAll(raw: CompositeTreeNode): Promise<boolean> {
```

Comment:
why to change it?

Reply:
It was code related to the previous implementation of tree from last month. I will remove this part.